test/MOI_wrapper.jl

```diff
@@ -37,19 +37,16 @@ function test_modificationtest(model, config)
 end
 
 function test_contlineartest(model, config)
-    return MOI.Test.contlineartest(
-        model,
-        config,
-        String[
-            # Upstream segfault. Reported: https://github.com/ERGO-Code/HiGHS/issues/448
-            # "linear8b",
-            # Upstream bug. Reported: https://github.com/ERGO-Code/HiGHS/issues/464
-            "linear8c",
-
-            # VariablePrimalStart not supported.
-            "partial_start",
-        ],
-    )
+    excludes = [
+        # VariablePrimalStart not supported.
+        "partial_start",
+    ]
+    if MOI.get(model, MOI.RawParameter("solver")) == "ipm"
+        # TODO(odow): investigate
+        push!(excludes, "linear8b")
```

Comment:
The IPM failed to generate certificates for these (simplex worked). Maybe we need to set other parameters?